src/qt/askpassphrasedialog.cpp

```diff
@@ -40,6 +40,8 @@ AskPassphraseDialog::AskPassphraseDialog(Mode _mode, QWidget *parent) :
     ui->passEdit2->installEventFilter(this);
     ui->passEdit3->installEventFilter(this);
 
+    ui->toggleShowPasswordButton->setIcon(QIcon(":/icons/eye"));
```

Comment:
This can be set in the `.ui`?

Reply:
It turns out it can.  Wasn't immediately apparent from the docs, but then I discovered QtCreator which made it much simpler to play around with.